### ISSUE TITLE
Enless loop of falling after teleporting to Argus

### ISSUE THREAD

**Issue by Dreda59** [AshamaneProject/AshamaneCore#476]
After clicking the Lightforged Beacon you teleport to argus but fall through the sky, before you hit the floor it teleports you back up and this continues on a loop. The only way to end this is to use a .tele command.

**Usefull Wowhead links:**

**Comment by Traesh** [comment]
Weird, it teleport you back because you fell throught the Vindicaar if it didn't load quick enough, when the Vindicaar finally show up you shoud stop falling

**Comment by Dreda59** [comment]
This was working as intended when this Scripts/Argus Fix falling of Vindicaar fix was added, I updated and added 

Core/BlackMarket Fix wrong GetSecondsRemaining() with simpler system,  
Core/Taxi Add Argus taxi to player (tempfix) and 
DB/Argus Add loots, quest starter/ender, npc_vendor, skinning, money … 

This is when it stopped working, so one of those 3 may have broken something. Have rebuilt today and still the same issue. Only the vindicaar matrix core seems to load.
